docs/sftt/sftt-api-spec.md

```diff
@@ -1873,6 +1873,43 @@ No request body.
 
 If the `site_id` specified is invalid.
 
+### Upload Site Image
+
+`POST api/v1/protected/sites/:site_id/upload_image`
+
+Used to update the image of a site. The `image` field of the specified entry in the `site_entries` table will 
```

Comment:
⛏️ The `image` field of the most recent entry in the `site_entries` table associated with the specified site will be updated.